packages/x-scheduler/src/internals/components/create-popover/createPopover.tsx

```diff
@@ -34,6 +35,8 @@ export function createPopover<TAnchorData>(config: CreatePopoverConfig) {
 
   function Provider(props: ProviderProps<TAnchorData>) {
     const { containerRef, children, renderPopover, onClose: onCloseProp, modal = true } = props;
+    const { isOpen: isScopeDialogOpen } = useScopeDialogContext();
```

Comment:
I'm not sure that this is the right place for this check. I think that `createPopover` should stay generic for any popover. 
The `MoreeventsPopover` does not care about the scope dialog for instance, only the `EventPopover` does. 

I think that we should be able to do this inside the `EventPopover.tsx` and try to manipulate the closing behavior with the `onCloseProp`. WDYT?